run.sh

```diff
@@ -56,11 +56,13 @@ if [ -z "$VPN_IPSEC_PSK" ] && [ -z "$VPN_USER" ] && [ -z "$VPN_PASSWORD" ]; then
     echo "VPN credentials not set by user. Generating random PSK and password..."
     VPN_IPSEC_PSK="$(LC_CTYPE=C tr -dc 'A-HJ-NPR-Za-km-z2-9' < /dev/urandom | head -c 16)"
     VPN_USER=vpnuser
-    VPN_PASSWORD="$(LC_CTYPE=C tr -dc 'A-HJ-NPR-Za-km-z2-9' < /dev/urandom | head -c 16)"
+    VPN_PASSWORD=group
```

Comment:
This line and the next line are incorrect. VPN password should be auto generated while the group name can have a default value.

Reply:
Oh, I didn't notice I changed the wrong line of code